doc/usage/restructuredtext/directives.rst

```diff
@@ -48,6 +48,11 @@ tables of contents.  The ``toctree`` directive is the central element.
    to the source directory.  A numeric ``maxdepth`` option may be given to
    indicate the depth of the tree; by default, all levels are included. [#]_
 
+   The representation of "TOC tree" is changed in each output format.  The
+   builders that output multiple files (ex. HTML) treat it as a collection of
+   hyperlinks.  On the other hand, the builders output a single file (ex. LaTeX,
```

Comment:
"On the other hand, the builders output a single file" -> "On the other hand, the builders *that* output a single file"

Reply:
Thanks!